app/src/main/java/com/tengu/sharetoclipboard/ShareToClipboardActivity.java

```diff
@@ -162,19 +178,10 @@ private String getSendTextString(Intent intent) {
         }
     }
 
-    @SuppressLint("NewApi")
     private void copyToClipboard(String clipboardText) {
-        int sdk = Build.VERSION.SDK_INT;
```

Comment:
This whole code is about supporting old Android versions, why do you think it's not needed?

Reply:
According to app/build.gradle our minimum supported version is API 15 (Ice Cream Sandwich). Honeycomb is API 11, which is older than that.

Reply:
I didn't remember we updated to ICS. Cool, will merge. Thanks!